packages/master-detail-layout/src/vaadin-master-detail-layout-transition-styles.js

```diff
@@ -0,0 +1,107 @@
+import { css } from 'lit';
```

Comment:
nit: let's also add copyright heading comment to this file.

Reply:
Thanks, added